integreat_cms/static/src/css/style.scss

```diff
@@ -596,3 +596,8 @@ datalist option {
         }
     }
 }
+
+/* Translation budget graph */
+.text-align-center {
+    text-align: center;
+}
```

Comment:
Then we don't need this

Reply:
Thanks, this is of course the far better solution, it somehow slipped my mind that we were using tailwind :D